src/renderer/css/components/TabBar.scss

```diff
@@ -0,0 +1,202 @@
+#main {
+  .TabBar {
+    border: none;
+    max-height: 40px;
+    flex: 1 1 auto;
+    display: flex !important;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+    -webkit-app-region: drag;
+    padding: 6px 8px;
+    background: inherit;
+  }
+
+  .TabBar-nonDrag {
+    flex-grow: 1;
+    -webkit-app-region: no-drag;
+  }
+
+  .TabBar .teamTabItem span {
+    flex: 0 1 auto;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    text-align: center;
+    -webkit-font-smoothing: auto;
+  }
+
+  .TabBar>li {
+    -webkit-app-region: no-drag;
+    -webkit-user-select: none;
+    min-width: 48px;
+    max-width: 224px;
+    position: relative;
+    margin: 0 2px;
+  }
+
+  .TabBar>li>a {
+    color: rgba(61,60,64,0.7);
+    font-family: "Open Sans", sans-serif;
+    font-weight: 600;
+    font-size: 12px;
+    letter-spacing: -0.2px;
+    transition: 0.3s;
+    line-height: 18px;
+    border-radius: 4px;
+    border: none !important;
+    padding: 0;
+  }
+
+  .TabBar.darkMode>li>a {
+    color: rgba(221,221,221,0.64);
+  }
+
+  .TabBar>li>a:hover {
+    background-color: rgba(255,255,255,0.56);
+    text-decoration: none;
+  }
+
+  .TabBar.darkMode>li>a:hover {
+    background-color: rgba(31, 31, 31, 0.56);
+  }
+
+  .TabBar>li>a:focus {
+    background-color: #fff;
+    color: #3d3c40;
+    text-decoration: none;
+  }
+
+  .TabBar>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close {
+      background: none;
+      border: none;
+      color: rgba(61,60,64,0.32);
+      margin-left: 6px;
+      padding: 0;
+  }
+
+  .TabBar>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close:hover {
+      color: #3d3c40;
+  }
+
+  .TabBar.darkMode>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close:hover {
+      color: #ddd;
+  }
+
+  .TabBar>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close>i::before {
+      margin: 0;
+  }
+
+  .TabBar.darkMode>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close {
+      color: rgba(221,221,221,0.32);
+  }
+
+  .TabBar.darkMode>li>a:focus {
+    background-color: #1f1f1f;
+    color: #ddd;
+  }
+
+  .TabBar>li>a>div.TabBar-tabSeperator {
```

Comment:
not sure if this is in the scope of this PR but nesting like this is probably what causes the style overrides. We might need to clean up desktop app styles. Regardless i would add a selector to either one of those elements, and remove two out the three.

Reply:
Yes these need a cleanup. Worth doing in a future PR.